rainforest/ml/rf_train.py

```diff
@@ -75,6 +75,13 @@ def main():
                       help="If set to 1 (default), the input parquet files (homogeneized tables) for the ml routines will be recomputed from the current database rows"+
                       "This takes a bit of time but is needed if you updated the database and want to use the new data in the training",
                       metavar="MODELS")
+    
+    parser.add_option("-l", "--logmlflow", 
```

Comment:
we need a flag to decide whether to log the model artifact, false by default?

Reply:
```
parser.add_option("-l", "--logmlflow", 
                  type="choice", 
                  choices=["none", "metrics", "all"],
                  dest="logmlflow", 
                  default="none",
                  help="Specify the logging mode for MLFlow. Choices are:" +
                       " 'none' (default, no logging), 'metrics' (log metrics only)," +
                       " or 'all' (log metrics and model)." +
                       " To log to a remote ML server, the environment variable MLFLOW_TRACKING_URI needs to be set.")
```
How about a choice like this?